synthesizer/sed.py

```diff
@@ -64,28 +71,39 @@ def __init__(self, lam, lnu=None, description=False):
         self.broadband_fluxes = None
 
     def __add__(self, second_sed):
+
+        """
+        Overide addition operator to allow two Sed objects to be added
+        together.
+        """
+
         if not np.array_equal(self._lam, second_sed._lam):
-            exceptions.InconsistentAddition("Wavelength grids must be identical")
+            exceptions.InconsistentAddition("Wavelength grids \
```

Comment:
None of these exceptions actually do anything since they are missing the `raise` keyword. i.e. `raise exceptions.InconsistentAddition(...)`.

Reply:
I have now fixed this.